metricbeat/docs/modules/sql.asciidoc

```diff
@@ -0,0 +1,44 @@
+////
+This file is generated! See scripts/mage/docs_collector.go
+////
+
+[[metricbeat-module-sql]]
+[role="xpack"]
+== sql module
+
+beta[]
+
+This is the sql module that fetches metrics from a SQL database. You can define driver, datasource and SQL query.
+
+
+
+
+[float]
+=== Example configuration
+
+The sql module supports the standard configuration options that are described
+in <<configuration-metricbeat>>. Here is an example configuration:
+
+[source,yaml]
+----
+metricbeat.modules:
+- module: sql
+  metricsets:
+    - query
+  period: 10s
+  hosts: ["localhost"]
+
+  driver: "postgres"
+  datasource: "user=myuser password=mypassword dbname=mydb sslmode=disable"
+  sql_query: "select now()"
+  ----
```

Comment:
There is still a problem here, it should be like that:
```suggestion
----
```

Reply:
Now I missed the make update...I think at least make docs it's fine now.

Reply:
This file looks good now, yes, could you run the `make update` again? Thanks!

Reply:
Done! :-)